### DISCUSSION TITLE
TubeZB Z-Wave Kit and ESPHome

### DISCUSSION THREAD

**Discussion by millercentral** [tube0013/tube_gateways#198] [Q&A]
I just received my TubeZB Z-Wave PoE kit and it is working great with ZwaveJS UI, thank you for an awesome product!

One question: The ESPHome add-on for HA discovered it. Should I adopt it into the ESPHome interface? 
On the plus side, it seems like it would be an easy way to monitor that it is online, and an easy way to apply updates.
My question/concern is: Doing this would prompt for updates every time ESPHome updates, is this needed or is it better to just leave a working device alone?

Thanks for any insights.

**Comment by tube0013** [comment]
I believe there is value in using the entity for the serial connection. Updating with each ESPHome release is not advised as there is no guarantee it will continue to work as expected. 

On the HA side I believe as of 2024.7 the ESPHome update entities are not enabled by default, but if it is you can just disable the update entity so not bugged by HA to do the update. And ESPHome has changed their dashboard to give less focus to updates.

Having said all that it is your device so ultimately it is your decision to update. Please know the original binaries I shipped on the unit are available here on GitHub if you ever need to roll back.
